test/test-util/src/mz.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright Materialize, Inc. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use std::time::Duration;
+
+use tokio_postgres::{Client, Error, NoTls, Row};
+
+/// Create and return a new PostgreSQL client, spawning off the connection
+/// object along the way.
+pub async fn client(user: &str, host: &str, port: u16) -> Result<Client, anyhow::Error> {
+    let (mz_client, conn) = tokio_postgres::Config::new()
+        .user(user)
+        .host(host)
+        .port(port)
+        .connect(NoTls)
+        .await?;
+
+    // The connection object performs the actual communication with the database,
+    // so spawn it off to run on its own.
+    tokio::spawn(async move {
+        if let Err(e) = conn.await {
+            log::info!("connection error: {}", e);
```

Comment:
Since it's a test utility, can we make this panic? `log::info!` is easy to ignore.